.travis.yml

```diff
@@ -14,35 +14,55 @@ php:
   - 7.2
   - nightly
 
-matrix:
+env:
+  global:
+    - deps=no
+
+before_install:
+  - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
+  - composer self-update
+  - if [ "$SYMFONY_VERSION" != "" ]; then jq "(.require, .\"require-dev\")|=(with_entries(if .key|test(\"^symfony/\") then .value|=\"${SYMFONY_VERSION}\" else . end))" composer.json|ex -sc 'wq!composer.json' /dev/stdin; fi;
+
+install:
+  - travis_retry composer update -n --prefer-dist
+
+script:
+  - ./vendor/bin/phpunit -v
+
+jobs:
   include:
     - php: 5.5
       env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
+      install:
+        - travis_retry composer update -n --prefer-lowest --prefer-stable --prefer-dist
+
     - php: 5.6
       env: SYMFONY_VERSION="2.8.*"
+
     - php: 7.0
       env: SYMFONY_VERSION="3.2.*"
```

Comment:
This should be `3.4` not `3.2`. 3.2 is not a LTS (and is EOLed)

Reply:
I think there is 3.4 already in separate stage. I didn't touch these, feel free to send PR.